CONTRIBUTING.md

```diff
@@ -39,12 +68,18 @@ Please use GitHub Discussions to submit feedback and engage with community [http
 - **Pass Tests**: Ensure all tests pass before submitting your PR.
 - **Be Open to Feedback**: We're all here to help each other improve, so please be open to feedback and ready to make adjustments.
 
-### Issues
+### Creating Issues
 
 - **Search First**: It helps to check if your problem or feature request has already been discussed before opening a new issue.
 - **Be Detailed**: When you open a new issue, providing as much detail as possible really helps. Feel free to use our templates for bugs and feature requests.
 - **Be Respectful**: Let's all be kind and considerate in our communication.
 
+### Solving Issues
+
+- Limit yourself to solving a maximum of four `good first issues`. Once you've reached this limit, consider tackling other types of issues.
+- Please work on only one issue at a time.
+- Please ask for assignee before working, and if there's no update for about a week on a particular issue, we'll remove the assignee.
```

Comment:
I'd like to add after the `we'll remove the assignee ... so that other people interested can be assigned`